src/FileOnQ.Imaging.Heif/IHeifImage.cs

```diff
@@ -2,9 +2,35 @@
 
 namespace FileOnQ.Imaging.Heif
 {
+	/// <summary>
+	/// A unprocessed Heif image.
```

Comment:
I don't think this is technically correct as heif images are processed from the camera, the format is just different than traditional formats such as JPEG. Maybe we can have this read

```
A High Efficiency Image Format (HEIF) image.
```

or something similar that describes it better.
```